fixtures/cassettes/collections_responses.yml

```diff
@@ -80656,6 +80656,32 @@ a31c03cb72c4be7f2e5973194936bdf6369055ee:
         aS5nc2ZjLm5hc2EuZ292L3JlY2lwZXMvP3E9cmVjaXBlLWNvb2tib29rIn1d
         fV19fQ==
     http_version: 
+43d3bd3bbc7174778467a8eee65dab91850abca2:
+  response:
+    status:
+      code: 401
+      message: 
+    headers:
+      access-control-allow-origin:
+      - "*"
+      cmr-request-id:
+      - 562e702c-2aa4-40e4-bad2-052986e6ea0d
+      content-type:
+      - application/json
+      date:
+      - Sun, 03 Dec 2017 19:06:20 GMT
+      server:
+      - nginx
+      strict-transport-security:
+      - max-age=31536000; includeSubDomains;
+      content-length:
+      - '47'
+      connection:
+      - Close
+    body:
+      encoding: UTF-8
+      string: '{"errors":["Token edsc-access does not exist"]}'
```

Comment:
This seems like a bad/undesirable recording. (There are two more below)

Reply:
I wonder if that's related to the wait_for_xhr bug?  Hrm.  I'll rerecord using e2e-services as a baseline, *after* the sleep/wait_for_xhr issue is fixed.